packages/contracts/deploy/update/to_v1.3.0/90_ManagingDAO.ts

```diff
@@ -5,32 +5,32 @@ import {getContractAddress} from '../../helpers';
 import {UPDATE_INFOS} from '../../../utils/updates';
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  console.log('\nUpdate ManagingDAO to new implemenation');
+  console.log('\nUpgrade the managing DAO to new implemenation');
 
   const daoFactoryAddress = await getContractAddress('DAOFactory', hre);
-  const daoImplementation = await DAOFactory__factory.connect(
+  const newDaoImplementation = await DAOFactory__factory.connect(
     daoFactoryAddress,
     hre.ethers.provider
   ).daoBase();
 
   const managingDAOAddress = await getContractAddress('managingDAO', hre);
-  const daoInterface = DAO__factory.connect(
+  const managingDAO = DAO__factory.connect(
     managingDAOAddress,
     hre.ethers.provider
   );
-  const upgradeTX = await daoInterface.populateTransaction.upgradeToAndCall(
-    daoImplementation,
-    daoInterface.interface.encodeFunctionData('initializeFrom', [[1, 0, 0], ''])
+  const upgradeTX = await managingDAO.populateTransaction.upgradeToAndCall(
+    newDaoImplementation,
+    managingDAO.interface.encodeFunctionData('initializeFrom', [[1, 0, 0], []])
   );
```

Comment:
i wonder if this DAO also need to upgrade itself,
why not do it later as well, perhaps via UI